File: src/my_project/easy_problems/from101to150/to_lower_case.py
from typing import List 

class Solution:
    def toLowerCase(self, s: str) -> str:
        answer: str = ""
        x = ""
        for letter in s:
            if ord(letter) <= 90 and ord(letter) >= 65:
                x = chr(ord(letter) + 32)
                answer += x
            else:
                answer += letter


        return answer 


solution = Solution()

print(solution.toLowerCase(s = "Hello"))

print(solution.toLowerCase(s = "here"))


print(solution.toLowerCase(s = "LOVELY"))

File: my_project_typescript/easy_problems_typescript/from1to50/toLowerCase.ts
function toLowerCase(s: string): string {
    let answer: string = ""
    let temp: string = ""

    for (let letter of s){
        if (letter.charCodeAt(0) <= 90 && letter.charCodeAt(0)>=65){
            temp = String.fromCharCode(letter.charCodeAt(0) + 32)
            answer += temp 
        }
        else{
            answer += letter
        }
        
    }

    return answer 
    
};


console.log(toLowerCase( "Hello"))

console.log(toLowerCase("here"))


console.log(toLowerCase("LOVELY"))
